add_to_app/flutter_module/lib/main.dart

```diff
@@ -148,6 +148,7 @@ class Contents extends StatelessWidget {
                   SizedBox(height: 16),
                   RaisedButton(
                     onPressed: () => SystemNavigator.pop(),
```

Comment:
Delete this line? The duplicate `onPressed:` is triggering a build failure on travis.

Reply:
oops, thanks